Makefile

```diff
@@ -0,0 +1,8 @@
+ORG=asahi
+NAME=builder
+
+build: ## Build docker image
+	@docker build -t $(ORG)/$(NAME) .
+
+installer: build ## Build asahi installer
+	@docker run --init --rm -v $(PWD):/asahi --platform linux/arm64 $(ORG)/$(NAME)
```

Comment:
Why is this using an arm64 image? Building the installer works just fine on x86_64. All development is being done on x86_64. This should be as simple as installing the requirements (including a cross-compiler) from the workflow and calling ./build.sh directly, no need for a Makefile or a Dockerfile.

Reply:
So arm images can be ran on x86.  Docker uses `binfmt_misc` which does a transparent arm emulation (using qemu I think) in this case the benefit is you get a docker images with the aarch64 gcc installed and ready to go with all the necessary deps etc (BUT since I then moved to using Github Actions full VMs the need for docker is unnecessary and those deps can be installed in the github workflow yml

Reply:
The point is everything is native x86, there is no need to run any arm64 software; only the *output* is arm64 software. Using arm64 images to build in qemu just makes everything an order of magnitude slower by introducing a binary translation layer for no reason; it doesn't make any sense. I designed the entire thing to build natively on x86.